packages/components/src/tabs/styles.ts

```diff
@@ -0,0 +1,234 @@
+import * as Ariakit from '@ariakit/react';
+import styled from '@emotion/styled';
+import { Icon } from '@wordpress/components';
```

Comment:
@mirka here's where `Tabs` needs `Icon` (just an FYI for when we make a local copy of it)

Reply:
I'll get my Icon PR merged first 👍